pandas/tests/io/json/test_normalize.py

```diff
@@ -460,3 +501,87 @@ def test_nonetype_multiple_levels(self):
             'location.country.state.town.info.y': -33.148521423339844,
             'location.country.state.town.info.z': 27.572303771972656}
         assert result == expected
+
+    def test_with_max_level_none(self):
+        # GH23843: Enhanced JSON normalize
+        data = [{
+            'CreatedBy': {'Name': 'User001'},
```

Comment:
it would be nice to parameterize these 3 tests, basically you put the max_level [None, 0, 1] along with the expected as parameters